### PR TITLE
Add the FieldSelect Component

### PR DESCRIPTION
**Pull request by Poltergeist**
Usage:
```
<FieldSelect name="protocol"
  value="http">
  <option value="">Select Protocol</option>
  <option value="http">HTTP</option>
  <option value="https">HTTPS</option>
</FieldSelect>
```
Will look like:
image

### CODE REVIEW

File: src/js/components/form/FieldSelect.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const FieldSelect = (props) => {
+  return (
+    <span className="form-control form-control-select">
```
Comment (mlunoe) [inline src/js/components/form/FieldSelect.js]: Couldn't this be directly on the select? That's what we do for other `form-control` elements

File: src/js/components/form/FieldSelect.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      <select {...props}>
+        {props.children}
+      </select>
+    </span>
+  );
+};
+
+FieldSelect.defaultProps = {
+  name: '',
```
Comment (mlunoe) [inline src/js/components/form/FieldSelect.js]: Let's not default name to anything

Comment (Poltergeist) [inline src/js/components/form/FieldSelect.js]: done.

**Comment by wavesoft** [comment]
Nice! ✨  :shipit:

**Pull request merged**